Library/BillingInfo.cs

```diff
@@ -401,6 +405,15 @@ internal override void WriteXml(XmlTextWriter xmlWriter)
                     xmlWriter.WriteElementString("external_hpp_type", ExternalHppType.Value.ToString().EnumNameToTransportCase());
 
                 }
+
+                if (!GatewayCode.IsNullOrEmpty())
+                {
+                    xmlWriter.WriteElementString("gateway_code", GatewayCode);
+                    xmlWriter.WriteElementString("gateway_token", GatewayToken);
+                    var card = CardType.ToString().EnumNameToTransportCase();

```

Comment:
Can we drop a comment here explaining why this is needed?

Reply:
Yup